src/main/controller/SvgFile.java

```diff
@@ -0,0 +1,18 @@
+package controller;
+
+import java.util.List;
+
+import model.Shapes;
+
+public interface SvgFile {	
+	
+	//if filename does not exist, create a new svg file
+	//if filename exists, then write content into it
+	public boolean writeSVGFile(String filename, String content);
+	
+	//add shapes to a svg file
+	public boolean addShapes(List<Shapes> listShapes);
```

Comment:
I would just say that here the result should actually be String (it will in fact be the content of the writeSVGFile, no?)